Vienna/Sources/Main window/OverlayStatusBar.swift

```diff
@@ -62,7 +62,7 @@ final class OverlayStatusBar: NSView {
     // MARK: Initialization
 
     init() {
-        super.init(frame: NSRect(x: 0, y: 0, width: 240, height: 26))
+        super.init(frame: NSRect(x: 0, y: 0, width: 4800, height: 26))
```

Comment:
AFAIK, this value is just a default. The sizing happens by Auto Layout. Was there a problem that this addresses?

Reply:
Yes, I add truncated addresses somewhere.

Reply:
I confirm that the problem is in Unified layout.